Automattic/wp-calypso
# Remove Quick Start upsell card from /help

#### Changes proposed in this Pull Request

* This PR removes the Quick Start upsell card from the support page shown under `/help`

#### Screenshots

##### Before

<img width="1231" alt="Screenshot 2021-11-15 at 12 28 16" src="https://user-images.githubusercontent.com/3376401/141766123-5a96c488-5ab2-4b77-8829-2331bc065d64.png">

##### After

<img width="1231" alt="Screenshot 2021-11-15 at 12 28 30" src="https://user-images.githubusercontent.com/3376401/141766099-e1e7b31c-1d4b-420a-8a1e-934358898845.png">

#### Testing instructions

* Run this branch locally or via the live branch (link is in [this comment](https://calypso.live/?image=registry.a8c.com/calypso/app:commit-37509a29e06ca5400209f02ebbee463ff3336479)).
* For a user where you don't have any purchased Quick Start sessions and you are permitted to purchase Quick Start sessions, navigate to `/help`.
* Verify that we don't show the Quick Start upsell card
